### PR TITLE
add: wishlist

### PR DESCRIPTION
**Pull request by user-64bit**

### CODE REVIEW

File: src/components/userProfile/userProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      href: '#',
+    },
+    {
+      label: 'Orders',
+      href: '#',
+    },
+    {
+        label: 'Wishlist',
+        href: '/wishlist',
+    },
```
Comment (ravi300601) [inline src/components/userProfile/userProfile.jsx]: remove extra space

Comment (0xuser64bit) [reply src/components/userProfile/userProfile.jsx]: Done :+1:

File: src/components/userProfile/userProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          <form action="#" method="POST">
+            <MenuItem>
+              <button
+                type="submit"
+                className="block w-full px-4 py-2 text-left text-sm text-gray-700 data-[focus]:bg-gray-100 data-[focus]:text-gray-900"
+              >
+                Sign out
+              </button>
+            </MenuItem>
+          </form>
```
Comment (ravi300601) [inline src/components/userProfile/userProfile.jsx]: Don't need to implement this function as this is already available for use

File: src/pages/wishlist/wishlist.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                                        </button>
+                                    </td>
+                                </tr>
+                            </>
+                        ))}
+
+                        {
+                            !wishListItems?.length && (
+                                <tr>
+                                    <td>No Items here..</td>
```
Comment (ravi300601) [inline src/pages/wishlist/wishlist.jsx]: display statement like
- "Oops! Your wishlist looks empty. Browse and add items to save for later."
- "You haven't added any items to your wishlist. Discover something special today!"
- "Nothing here yet! Explore our collection and add items to your wishlist."

File: src/pages/wishlist/wishlist.jsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                                </tr>
+                            )
+                        }
+                    </tbody>
+                </table>
+            </div>
+
```
Comment (ravi300601) [inline src/pages/wishlist/wishlist.jsx]: remove extra line

**Review by ravi300601 — CHANGES_REQUESTED** [review]

File: src/components/userProfile/userProfile.jsx (hunk 2 above)
Comment (0xuser64bit) [reply src/components/userProfile/userProfile.jsx]: Done :+1: 
.thanks

**Review by ravi300601 — APPROVED** [review]

**Comment by ravi300601** [comment]
LGTM :rocket: 
Thanks for your work.

**Pull request merged**